hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/RecordListBuilder.java

```diff
@@ -385,6 +385,25 @@ public void revertChildrenOf(@NonNull final SingleTransactionRecordBuilderImpl r
         }
     }
 
+    /**
+     * Reverts or removes the last child transaction on which the revert() method was called.
+     * <p> If the child record is REVERSIBLE, then the status is set to {@link ResponseCodeEnum#REVERTED_SUCCESS}.
+     * If the child record is REMOVABLE, then the record is removed from the list.
+     */
+    public void revertLastChildOnly() {
```

Comment:
Hey @Neeharika-Sompalli and @tinker-michaelj, I need your opinion on this. Is it okay to revert only the last child record in this case, considering the explanation in the other comment? If we anticipate that the child calling revert might not always be the last one, perhaps we can keep track of the children or the context?